### PR TITLE
Documentation of our project

### PR DESCRIPTION
**Pull request by Nikhil-Choudhary**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,150 @@
...
+
+### plane mesh made from wire
+ ![Project Images](https://github.com/Nikhil-Choudhary/VR-Flight-Simulator/blob/master/Photos%20And%20Videos/photos/plane.jpeg)
+
+### Project Mechanical design
+ ![Project Images](https://github.com/Nikhil-Choudhary/VR-Flight-Simulator/blob/master/Photos%20And%20Videos/photos/Project%20Mechanical%20design.jpeg)
+
+### Game made on unity
+![Project Images](https://github.com/Nikhil-Choudhary/VR-Flight-Simulator/blob/master/Photos%20And%20Videos/photos/Game.jpeg)
+***
```
Comment (agshubhs) [inline README.md]: *** No need of these lines after each section

File: README.md [hunk 2]
```diff
@@ -1,2 +1,150 @@
...
+## MOTIVATION:
+
+Our motivation for doing this project was the creativity of the idea and our fascination with the end result.
+***
+## MECHANICAL ASPECT OF DESIGN:
+
+Our mechanical parts consists of
+
+1. joystick
```
Comment (agshubhs) [inline README.md]: First letter should be capital

File: README.md [hunk 3]
```diff
@@ -1,2 +1,150 @@
...
+2. throttle
+
+Explaination :-
```
Comment (agshubhs) [inline README.md]: The spelling of Explanation.

Remove the upper line write the name and description simultaneously.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,150 @@
...
+	
+	this device is basically used for controling the speed of aeroplane.
+	
+Both the devices are made from basic components such as ply , a man made springs , steel channels , ball bearings , and joining components
+***
+## Electronic ASPECT OF DESIGN:
+
+Electronics Consist of use of
+
+1. Ultrasonic sensor
```
Comment (agshubhs) [inline README.md]: t should be written like this and the same as for mechanical aspect-
1. **Ultrasonic Sensor** - for giving pitch to plane
2. and so on....

File: README.md [hunk 5]
```diff
@@ -1,2 +1,150 @@
...
+
+	for detecting Rocket shooting
+
+5. Esp module
+
+	for receiving data from sensors and sending it to unity	
+***
+## Complete Working of Project
+
+what we are doing is that
```
Comment (agshubhs) [inline README.md]: No need of this line

File: README.md [hunk 6]
```diff
@@ -1,2 +1,150 @@
...
+	
+4. Bump Sensor
+
+	for detecting Rocket shooting
+
+5. Esp module
+
+	for receiving data from sensors and sending it to unity	
+***
+## Complete Working of Project
```
Comment (agshubhs) [inline README.md]: Title should be 
## Working

File: README.md [hunk 7]
```diff
@@ -1,2 +1,150 @@
...
+
+what we are doing is that
+* taking input from 4 sensors(in form of nos.) and sending this input to unity software through esp module
```
Comment (agshubhs) [inline README.md]: First letter should be capital

File: README.md [hunk 8]
```diff
@@ -1,2 +1,150 @@
...
+* after receiving data we are scaling it in unity to make it in suitable form and sending it to our game
+* then we are using a software called trinus vr to project computer screen onto our mobile in form of 3d environment by making pc as server and mobile as receiver
+* This is it. Syncing sensor input and input in unity was very crucial.
+ ***
+## COST STRUCTURE:
+
```
Comment (agshubhs) [inline README.md:74]: Make table for the cost structure

File: README.md [hunk 9]
```diff
@@ -1,2 +1,150 @@
...
+ 
+2. Electrical
+	* esp8266 module       : rs 500
+	* ultrasonic sensor    : rs 100
+	* bump sensor          : rs 20
+	* potentiometer sensor : rs 50
+***
+## APPLICATIONS:
+
+It can used for real life pilot training. It can also be used as a game for mass selling.
```
Comment (agshubhs) [inline README.md]: Write this as a numbered List.
1. Can be used for real life pilot training.
2. Can be used for mass selling.

File: README.md [hunk 10]
```diff
@@ -1,2 +1,150 @@
...
+	* ultrasonic sensor    : rs 100
+	* bump sensor          : rs 20
+	* potentiometer sensor : rs 50
+***
+## APPLICATIONS:
+
+It can used for real life pilot training. It can also be used as a game for mass selling. 
+***
+## LIMITATIONS:
+
```
Comment (agshubhs) [inline README.md:79]: Write this as numbered List

File: README.md [hunk 11]
```diff
@@ -1,2 +1,150 @@
...
+There was a slight noticable delay in rotation of joystick and its effect on screen, due to a long method of transmission of signal.
+The system also failed on end days so its durablity can be improved.
+***
+## FUTURE IMPROVMENTS:
+
+The delay of joystick rotation to its result can be decreased by using a short way of giving a signal to software.
+Some software improvments can be done such as adding of distance from mountain (or building) in front, or if the plane is losing stability.
+***
```
Comment (agshubhs) [inline README.md]: Remove all the unnecessary horizontal lines.

File: README.md (hunk 1 above)
Comment (agshubhs) [inline README.md]: Try to use images through the readme file. Do not use all in the beginning.

**Comment by agshubhs** [comment]
@Nikhil-Choudhary 
Changes are needed in readme file

**Comment by Nikhil-Choudhary** [comment]
I will make these changes in the evening

**Comment by Nikhil-Choudhary** [comment]
Sir may you please review the readme file. I have made stated changes

File: README.md [hunk 12]
```diff
@@ -1,2 +1,138 @@
 # VR-Flight-Simulator
 Srishti 2019
+
+### Project
+![Project Images](https://github.com/Nikhil-Choudhary/VR-Flight-Simulator/blob/master/Photos%20And%20Videos/photos/Game.jpeg)
+
+## ABSTRACT:
+
+The project we made is known as vr flight simulator.It is a simulation of real life plane experience.
```
Comment (agshubhs) [inline README.md]: Here VR should be capital

File: README.md [hunk 13]
```diff
@@ -1,2 +1,138 @@
...
+respective motion in game. The throttles went front and backwards to accelerate and decelerate the plane
+respectively.A game on pc was developed by us which contains a plane and some objects which can be shot.
+
+## MOTIVATION:
+
+Our motivation for doing this project was the creativity of the idea and our fascination with the end result.
+
+## MECHANICAL ASPECT OF DESIGN:
+
+### Project Mechanical Design
```
Comment (agshubhs) [inline README.md]: Write only Mechanical Design instead of PRoject mechanical Design

File: README.md [hunk 14]
```diff
@@ -1,2 +1,138 @@
...
+ ![Project Images](https://github.com/Nikhil-Choudhary/VR-Flight-Simulator/blob/master/Photos%20And%20Videos/photos/Project%20Mechanical%20design.jpeg)
+ 
+ ### Plane Mesh Made From Wire
+ ![Project Images](https://github.com/Nikhil-Choudhary/VR-Flight-Simulator/blob/master/Photos%20And%20Videos/photos/plane.jpeg)
+
+Our mechanical parts consists of
+	
+1. Joystick	-This device is used for controlling the aeroplane so that user experiences reality ( gives us a real experience ), with the help of this device we tried to make environment like that of aeroplane .
```
Comment (agshubhs) [inline README.md]: Joystick should be in bold

File: README.md [hunk 15]
```diff
@@ -1,2 +1,138 @@
...
+
```
Comment (agshubhs) [inline README.md]: Remove this  line space

File: README.md [hunk 16]
```diff
@@ -1,2 +1,138 @@
...
+2. Throttle 	-This device is basically used for controling the speed of aeroplane.
+	
+Both the devices are made from basic components such as ply , a man made springs , steel channels , ball bearings , and joining components
+
+## Electronic ASPECT OF DESIGN:
+
+Electronics Consist of use of
+
+1. Ultrasonic sensor	-for giving pitch to plane
+
```
Comment (agshubhs) [inline README.md]: Remove the linespace from the numbered list

File: README.md [hunk 17]
```diff
@@ -1,2 +1,138 @@
...
+3. Potentiometer	-for detecting throttle
+	
+4. Bump Sensor	-for detecting Rocket shooting
+
+5. Esp module	-for receiving data from sensors and sending it to unity	
+
+## Working
+
+### while paying
```
Comment (agshubhs) [inline README.md]: it should be playing not paying

**Comment by agshubhs** [comment]
@Nikhil-Choudhary  All the subsection title either should be capital or normal. In your readme some are written in capital and some are normal so maintain the uniformity.

**Comment by Nikhil-Choudhary** [comment]
Sir, may you please review this again. I have made mentioned changes and some change from my own.

**Comment by agshubhs** [comment]
@Nikhil-Choudhary 
1. The spelling of Mechanical Design in the folder is wrong.
2. Unnecessary files are present inside the Mechanical design folder. Remove them if they don't have any link with this project.
3. Use HTML syntax of the images to resize the image.

File: README.md [hunk 18]
```diff
@@ -1,2 +1,132 @@
...
+* Taking input from 4 sensors(in form of nos.) and sending this input to unity software through esp module
+* After receiving data we are scaling it in unity to make it in suitable form and sending it to our game
+* Then we are using a software called trinus vr to project computer screen onto our mobile in form of 3d environment by making pc as server and mobile as receiver
+* This is it. Syncing sensor input and input in unity was very crucial.
+ 
+## COST STRUCTURE
+
+1. **Mechanical** 
+
+	|  Component |Cost  RS. |
```
Comment (agshubhs) [inline README.md]: 1. Write Cost(INR) in place of the Cost Rs. in both the table and make a single table.
2. Add total row in the table also.

**Comment by agshubhs** [comment]
@Nikhil-Choudhary 
1. The spelling of the Mechanical Design Folder is incorrect. Correct it.
2. Delete unnecessary solid works files which are not used in your final projects.
3. Folder name should be **Images and Videos**, not **photos and Videos**.

**Pull request closed without merging**